src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/LaunchSettingsProvider.cs

```diff
@@ -663,15 +681,21 @@ protected Task HandleLaunchSettingsFileChangedAsync()
                 // throttle.
                 if (!FileManager.FileExists(fileName) || FileManager.LastFileWriteTime(fileName) != LastSettingsFileSyncTime)
                 {
-                    return FileChangeScheduler.ScheduleAsyncTask(token =>
+                    return FileChangeScheduler?.ScheduleAsyncTask(token =>
                     {
                         if (token.IsCancellationRequested)
                         {
                             return Task.CompletedTask;
                         }
 
+                        SequentialTaskExecutor? sequentialTaskQueue = _sequentialTaskQueue;
+                        if (sequentialTaskQueue == null)
+                        {
+                            throw new ObjectDisposedException(nameof(LaunchSettingsProvider));
+                        }
+
                         // Updates need to be sequenced
-                        return _sequentialTaskQueue.ExecuteTask(() => UpdateProfilesAsync(null));
+                        return sequentialTaskQueue.ExecuteTask((() => UpdateProfilesAsync(null)));
```

Comment:
>((() [](start = 62, length = 4)

Are the extra parentheses necessary?

Reply:
Nope - artifact of the Expand/Reduce API going wrong. Fixed